etn-ccis/blui-react-workflows
# Render children in RegistrationContextProvider and update example2 dependencies

Fixes BLUI-4143.



#### Changes proposed in this Pull Request:

- Render children in RegistrationContextProvider and update example2 dependencies



#### To Test:

- This should build with `yarn start:example2`
- The language buttons won't change as they aren't really hooked up any longer. I will fix that in an upcoming PR, but you can test the language by modifying the i18n language key in the application's local storage data. 



#### Any specific feedback you are looking for?

-
